src/components/v2/TokenTextField/FormikTokenTextField.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useField } from 'formik';
+import { TokenTextField, ITokenTextFieldProps } from '.';
+
+interface IFormikTokenTextField extends Omit<ITokenTextFieldProps, 'name' | 'onChange' | 'value'> {
+  name: string;
+  errorCodes?: string[];
+}
+
+export const FormikTokenTextField = ({ name, errorCodes = [], ...rest }: IFormikTokenTextField) => {
+  const [{ value, onBlur }, { error }, { setValue }] = useField(name); // eslint-disable-line  @typescript-eslint/no-unused-vars
+  const onChange = (val: string) => {
+    setValue(val);
+  };
+  return (
+    <TokenTextField
+      value={value}
+      onChange={onChange}
+      onBlur={onBlur}
+      hasError={!!(error && errorCodes.includes(error))}
+      {...rest}
+    />
+  );
+};
```

Comment:
We are using these files to have base interactive form components that conform to our UX form pattern. The advantage of this is that if we implement a form that breaks our UX pattern it will be obvious because we'd have to either implement an input and buttons directly or create a new component